domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/test/kotlin/ch/tutteli/atrium/assertions/filesystem/SymbolicLinkResolvingSpec.kt

```diff
@@ -174,41 +188,79 @@ object SymbolicLinkResolvingSpec : Spek({
                         { isSameAs(testAssertion) }
                     )
             }
+
+            it("does not assume a link loop even if the same link appears multiple times") {
+                val testdir = tempFolder.newFolder("multi-non-loop").toRealPath()
+                val barLink = testdir.createSymbolicLink(testdir.resolve("bar"))
+                val target = testdir.resolve("target").createFile()
+                val testLink = barLink.resolve(barLink.fileName).resolve(barLink.fileName).resolve("target")
```

Comment:
```suggestion
                val testLink = barLink.resolve(barLink.fileName).resolve(barLink.fileName).resolve(target.fileName)
```
Should also fix the build

Reply:
Ah, thanks!